alternator/executor.cc

```diff
@@ -2954,24 +2979,60 @@ future<executor::request_return_type> executor::batch_write_item(client_state& c
                 co_return api_error::validation(fmt::format("Unknown BatchWriteItem request type: {}", r_name));
             }
         }
-        per_table_wcu.emplace_back(per_table_stats, wcu_delete_units, wcu_put_units);
-        if (should_add_wcu) {
-            rjson::value entry = rjson::empty_object();
-            rjson::add(entry, "TableName", rjson::from_string(rjson::to_string_view(it->name)));
-            rjson::add(entry, "CapacityUnits", wcu_delete_units + wcu_put_units);
-            rjson::push_back(consumed_capacity, std::move(entry));
-        }
+        per_table_wcu.emplace_back(per_table_stats, schema, rjson::to_string_view(it->name));
```

Comment:
If I understand correctly, it->name is the table name. Why do you need both the schema (which has cf_name() in it, right?) and it->name?
(I still don't understand what you're changing here at all, but even if you don't, I don't know why you need to save both schema and table name).